pom.xml

```diff
@@ -8,5 +8,28 @@
     <artifactId>vinyl-project</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+    </properties>
+
```

Comment:
Why 11, and not 15?

Reply:
@AlexKievDeveloper which version we use in this project: 11 or 15?

Reply:
11 because it is LTS (long term support version == safety)

> @AlexKievDeveloper which version we use in this project: 11 or 15?

11 because it is LTS (long term support version == safety)